tests/test_transformer.py

```diff
@@ -71,6 +71,117 @@ def test_from_wkt_multipolygon(simplify_transformer):
     ]
 
 
+def test_from_geo_json(simplify_transformer):
+    # Clockwise polygon remains clockwise
+    assert simplify_transformer.from_geo_json({
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [50, 20],
+                [50, 21],
+                [51, 21],
+                [51, 20],
+                [50, 20],
+            ],
+        ],
+    }) == [
+        Polygon([
+            (50.0, 20.0),
+            (50.0, 21.0),
+            (51.0, 21.0),
+            (51.0, 20.0),
+            (50.0, 20.0),
+        ]),
+    ]
+    assert simplify_transformer.from_geo_json({
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [1, 1],
+                [2, 1],
+                [1, 2],
+                [1, 1],
+            ],
+        ],
+    }) == [
+        Polygon([
+            (1, 1),
+            (2, 1),
+            (1, 2),
+            (1, 1),
+        ]),
+    ]
+    # Duplicate point is removed
+    assert simplify_transformer.from_geo_json({
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [50.0, 20.0],
+                [50.0, 21.0],
+                [51.0, 21.0],
+                [51.0, 20.0],
+                [50.0, 20.0],
+                [50.0, 20.0],
+            ],
+        ],
+    }) == [
+        Polygon(
+            [
+                (50.0, 20.0),
+                (50.0, 21.0),
+                (51.0, 21.0),
+                (51.0, 20.0),
+                (50.0, 20.0),
+            ],
+        ),
+    ]
+
+
+def test_from_geo_json_multipolygon(simplify_transformer):
+    assert simplify_transformer.from_geo_json(
+        {
+            "type": "MultiPolygon",
+            "coordinates": [
+                [
+                    [
+                        [30, 20],
+                        [45, 40],
+                        [10, 40],
+                        [30, 20],
+                    ],
+                ],
+                [
+                    [
+                        [15, 5],
+                        [40, 10],
+                        [10, 20],
+                        [5, 10],
+                        [15, 5],
+                    ],
+                ],
+            ],
+        },
+    ) == [
+       Polygon(
+           [
+               (30.0, 20.0),
+               (45.0, 40.0),
+               (10.0, 40.0),
+               (30.0, 20.0),
+           ],
+       ),
+       Polygon(
+           [
+               (15.0, 5.0),
+               (40.0, 10.0),
+               (10.0, 20.0),
+               (5.0, 10.0),
+               (15.0, 5.0),
+           ],
+       ),
+   ]
```

Comment:
this is under indented by 1 space
```suggestion
    ]
```